docs/source/upgrade-guide/index.md

```diff
@@ -48,6 +48,56 @@ You should take the following actions for your Volto 19 projects.
 
 If you can't upgrade immediately, you may continue to run Volto 19 on Node.js 20 at your own risk, but be aware that issues specific to Node.js 20 will not be fixed in the Volto core CI or releases.
 
+(replace-razzle-with-volto-razzle)=
+
+### Replace `razzle` with `@volto/razzle` (fork)
+```{versionchanged} Volto 19.0.0-alpha.10
+```
+
+`@volto/razzle` is a fork of the upstream `razzle` package that contains Volto-specific fixes and patches.
+We recommend using `@volto/razzle` in your Volto 19 projects either when you need the Volto-compatible build behavior or when we provide temporary patches that are not yet merged upstream.
+
+Recommended steps to switch:
+
+1.  Remove the existing `razzle` dependency and install `@volto/razzle` as a development dependency.
+    For example, with `pnpm`:
+
+    ```shell
+    pnpm remove razzle
+    pnpm add -D @volto/razzle
+    ```
+
+2.  Update your {file}`package.json`'s `devDependencies` entry.
+    For example:
+
+    ```diff
+      "devDependencies": {
+    -    "razzle": "4.2.18",
+    +    "@volto/razzle": "x.x.x",
```

Comment:
Would this be necessary in projects/add-ons? I don't think so.

Reply:
I was thinking about using the old Volto project generator. But I don't think we need to worry about that anymore. I'll remove it.